generators/app/templates/src/StarterKit/Program.cs

```diff
@@ -1,44 +1,71 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
 using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
-using System;
-using System.IO;
+using Serilog;
+using Serilog.Events;
+using StarterKit.Shared.Constants;
+using StarterKit.DataAccess.Options;
+using StarterKit.Startup;
 
 namespace StarterKit
 {
   public class Program
   {
-    public static void Main(string[] args)
+    public static async Task Main(string[] args)
     {
-      CreateWebHostBuilder(args).Build().Run();
+      // Get a configuration up and running
+      var configPath = Path.Combine(Directory.GetCurrentDirectory(), JsonFilesKey.JsonFilesPath);
+      var loggingConfig = new ConfigurationBuilder().SetBasePath(configPath).AddJsonFile(JsonFilesKey.LoggingJson).Build();
+
+      // Set up our preliminary logger
+      Log.Logger = new LoggerConfiguration()
+        .MinimumLevel.Debug()
+        .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
+        .ReadFrom.Configuration(loggingConfig, "Logging")
+        .Enrich.FromLogContext()
+        .CreateLogger();
```

Comment:
misschien deze al in commentaar toevoegen?
.WriteTo.Console(new RenderedCompactJsonFormatter()))
Deze geeft de optie om de logs te laten oppikken door een extern systeem zoals FileBeat.
Ook geeft dit kleinere logfiles waardoor het netwerk en de randsystemen minder belast zullen worden.